e2etest/pkg/frr/container/container.go

```diff
@@ -365,3 +374,30 @@ func containerIsRunning(containerName string) error {
 
 	return nil
 }
+
+func SetupP2PPeer(frrImage string, node corev1.Node) (*FRR, error) {
```

Comment:
seems the functions care only about the container's name, not the entire corev1.Node. wdyt about
```go
CreateP2PPeerFor(containerName, frrImage string)?
```

Reply:
done